insertBaseData.sql

```diff
@@ -884,11 +884,50 @@ INSERT INTO noteRelatedObjects (noteUuid, relatedObjectType, relatedObjectUuid)
 	WHERE r.text LIKE 'Today%';
 
 -- LEAVE THIS AS LAST STATEMENT
--- Truncate all the dates on reports to dates that could have been generated by
+-- Truncate all the dates (on reports etc.) to dates that could have been generated by
 -- Java (millisecond precision) rather than by the database itself (microsecond precision)
 -- Note: keep SET on same line as UPDATE so mssql2sqlite.sh works correctly
 UPDATE reports SET
     createdAt=cast(createdAt as datetime2(3)),
     updatedAt=cast(updatedAt as datetime2(3)),
+    releasedAt=cast(updatedAt as datetime2(3)),
     engagementDate=cast(engagementDate as datetime2(0))
   ;
+UPDATE people SET
+    createdAt=cast(createdAt as datetime2(3)),
+    updatedAt=cast(updatedAt as datetime2(3)),
+    endOfTourDate=cast(endOfTourDate as datetime2(0))
+  ;
+UPDATE positions SET
+    createdAt=cast(createdAt as datetime2(3)),
+    updatedAt=cast(updatedAt as datetime2(3))
+  ;
+UPDATE peoplePositions SET
+    createdAt=cast(createdAt as datetime2(3))
+  ;
+UPDATE organizations SET
+    createdAt=cast(createdAt as datetime2(3)),
+    updatedAt=cast(updatedAt as datetime2(3))
+  ;
+UPDATE tasks SET
+    createdAt=cast(createdAt as datetime2(3)),
+    updatedAt=cast(updatedAt as datetime2(3)),
+    plannedCompletion=cast(updatedAt as datetime2(3)),
```

Comment:
Shouldn't we use  datetime2(0) for planned and projected completion?

Reply:
Strictly speaking, yes. I only added these lines as an afterthought, since they are *not used* in `insertBaseData.sql`, but it does no harm to change it, indeed.